pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/SectionBlockDao.java

```diff
@@ -338,13 +338,14 @@ default void insertTabularBlock(long activityId, TabularBlockDef block, long rev
 
         for (int row = 0; row < block.getRows().size(); row++) {
             for (int column = 0; column < block.getRows().get(row).getQuestions().size(); column++) {
-                final var question = block.get(row, column);
-                if (question == null) {
+                final var questionBlockDef = block.get(row, column);
+                if (questionBlockDef == null || questionBlockDef.getQuestion() == null) {
                     continue;
                 }
-
-                getQuestionDao().insertQuestionByType(activityId, question, revisionId);
-                getJdbiBlockTabular().insertQuestion(tabularId, question.getQuestionId(), column, row);
+                Integer columnSpan = questionBlockDef.getColumnSpan();
```

Comment:
Why do you need a separate variable? It could be inlined.

```
getJdbiBlockTabular().insertQuestion(tabularId, questionBlockDef.getQuestion().getQuestionId(),
                        column, row, questionBlockDef.getColumnSpan());
```